### PR TITLE
New RSS Reader App with Brand Fetching Features

### PR DESCRIPTION
**Pull request by salmanfarisvp**
- [X]  Update the colours as per the brand's primary and secondary colours
- [X] Update the in app logo as per the brand logo. 
- [X] Override time zone
- [X]  Override locale setting for region languages in dates.
- [X] Fetch RSS data and display
- [X] Support Standard Screen Sizes - As per [Supported Screen Resolutions](https://github.com/Screenly/Playground/blob/master/docs/resolutions.md) 
- [X] Tracks/capture errors using [Sentry](https://www.sentry.io/)
- [X] Added theme support.
- [X] Added Signal Ready
- [ ] QA is done by at least two or more members.

### CODE REVIEW

**Comment by salmanfarisvp** [comment]
image

- [X] Brand fetching UI is complete 
- [ ] Working on displaying RSS feed in the appropriate container

**Comment by vpetersson** [comment]
Should this be two columns? Just having one big column to the left would make more sense, no?

**Comment by salmanfarisvp** [comment]
@vpetersson , the two columns to showcase the two different news feed. One can also work, but as per the design there is two. Attaching the mockup design below for reference .  

image

**Comment by vpetersson** [comment]
So we only show two news elements from the RSS? That is a very different behavior than we had before. We also have a number of settings, including [limit](https://github.com/Screenly/Playground/blob/master/edge-apps/rss-reader/screenly.yml#L29), that are no longer applicable then.

**Comment by salmanfarisvp** [comment]
Yes, as per the current design we have two content element.  I can check with @L4cko and implement one column as you mentioned or other method that align with previous method. 

here is the current app for reference. 
image

**Comment by nicomiguelino** [comment]
@salmanfarisvp, please take a look at my temporary PR #195. Feel free to use it as a guide in making changes in this pull request.

**Comment by salmanfarisvp** [comment]
@vpetersson , Adnan designed a new UI by adding more feeds. What do you think? We can choose one design and I can work on it.  

cc @nicomiguelino 

image

**Comment by vpetersson** [comment]
I like v2

**Comment by salmanfarisvp** [comment]
> I like v2

Thanks for the swift feedback.

**Comment by salmanfarisvp** [comment]
Update 

- [X] Able to fetch the RSS feed and display correclty on the App

ToDo
- [ ] Fixing the UI issues with different screen layout  -> issue with portrait mode. 

image

File: edge-apps/rss-reader/index.html [hunk 1]
```diff
@@ -1,110 +1,82 @@
...
+						</div>
+					</div>
+				</div>
+			</div>
+			<div class="row-container">
+				<div class="secondary-card">
+					<div class="secondary-card date-card">
+						<span class="date-text">MAY</span>
+						<span class="date-number">26</span>
+					</div>
```
Comment (iamplugged) [inline edge-apps/rss-reader/index.html:63]: Do we need to show the defaults here before replacement?

Comment (salmanfarisvp) [reply edge-apps/rss-reader/index.html:63]: Adding a default value is not mandatory but is useful when running an emulator without values. I can remove it if you'd like.

File: edge-apps/rss-reader/static/js/main.js [hunk 2]
```diff
@@ -133,3 +131,191 @@ const getRssData = function () {
...
+      const pubDate0 = moment(localStorage.getItem('pubDate-0')).tz(timezone).locale(locale).format(format)
+      const pubDate1 = moment(localStorage.getItem('pubDate-1')).tz(timezone).locale(locale).format(format)
+      const pubDate2 = moment(localStorage.getItem('pubDate-2')).tz(timezone).locale(locale).format(format)
+      const pubDate3 = moment(localStorage.getItem('pubDate-3')).tz(timezone).locale(locale).format(format)
+
+      document.querySelector('.content-timestamp-0').innerText = pubDate0
+      document.querySelector('.content-timestamp-1').innerText = pubDate1
+      document.querySelector('.content-timestamp-2').innerText = pubDate2
+      document.querySelector('.content-timestamp-3').innerText = pubDate3
+    }
```
Comment (iamplugged) [inline edge-apps/rss-reader/static/js/main.js:209]: This can be generalized. A loop can be used.

File: edge-apps/rss-reader/static/js/main.js [hunk 3]
```diff
@@ -133,3 +131,191 @@ const getRssData = function () {
...
+
+  await timeAndLocale()
+
+  // constant colors
+  const tertiaryColor = '#FFFFFF'
+  const backgroundColor = '#C9CDD0'
+
+  // Theme Selection
+  const theme = screenly.settings.theme ? screenly.settings.theme : 'light'
+
```
Comment (iamplugged) [inline edge-apps/rss-reader/static/js/main.js:222]:
```suggestion
  const theme = screenly.settings.theme || 'light'

```

File: edge-apps/rss-reader/index.html (hunk 1 above)
Comment (salmanfarisvp) [reply edge-apps/rss-reader/index.html]: Fixed, removed default values.

File: edge-apps/rss-reader/static/js/main.js (hunk 2 above)
Comment (salmanfarisvp) [reply edge-apps/rss-reader/static/js/main.js]: Thanks, Using a loop now. 

```
  const format = moment.localeData(locale).longDateFormat('LT').includes('A') ? 'ddd MMM DD YYYY hh:mm A' : 'ddd MMM DD YYYY HH:mm'
  const pubDates = ['pubDate-0', 'pubDate-1', 'pubDate-2', 'pubDate-3'].map(dateKey => {
        return moment(localStorage.getItem(dateKey)).tz(timezone).locale(locale).format(format)
      })

 pubDates.forEach((pubDate, index) => {
        document.querySelector(`.content-timestamp-${index}`).innerText = pubDate
      })
```

File: edge-apps/rss-reader/static/js/main.js [hunk 4]
```diff
@@ -133,3 +132,189 @@ const getRssData = function () {
...
+    // If CORS fails, try the fallback URL
+    try {
+      await fetchImage(fallbackUrl)
+    } catch (fallbackError) {
+      // If fallback fails, use the default logo
+      imgElement.src = defaultLogo
+    }
+  }
+
+  document.querySelectorAll('.rss-tittle').forEach(element => {
```
Comment (iamplugged) [inline edge-apps/rss-reader/static/js/main.js:315]: typo here - rss-tittle

**Review by iamplugged — APPROVED** [review]

**Pull request merged**